### ISSUE TITLE
torch.nn.MultiMarginLoss does not have doc for shape

### ISSUE THREAD

**Issue by cccntu** [pytorch/pytorch#63747]
## 📚 Documentation

This is the doc with missing documentation for input/output shape:
https://pytorch.org/docs/stable/generated/torch.nn.MultiMarginLoss.html

This is another similar class, which has documentation for input/output shape:
https://pytorch.org/docs/stable/generated/torch.nn.MultiLabelMarginLoss.html

**Comment by ozancaglayan** [comment]
I came to open this issue as well, thanks for doing this. There's also a minor typo in the latex equation for the weighted loss:
`\text{loss}(x, y) = \frac{\sum_i \max(0, w[y] * (\text{margin} - x[y] + x[i]))^p)}{\text{x.size}(0)}`

essentially, the last `)` after `^p` seems to be an extra parenthesis.
